src/EventSubscriber/SendFailedMessageForRetryListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MessengerHeartbeatBundle\EventSubscriber;
+
+use Answear\MessengerHeartbeatBundle\Exception\RabbitMQTransportException;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Messenger\Event\WorkerMessageFailedEvent;
+use Symfony\Component\Messenger\EventListener\SendFailedMessageForRetryListener as SymfonySendFailedMessageForRetryListener;
+use Symfony\Component\Messenger\Stamp\ErrorDetailsStamp;
+
+class SendFailedMessageForRetryListener implements EventSubscriberInterface
+{
+    public function __construct(
+        private SymfonySendFailedMessageForRetryListener $decoratedSubscriber,
+        private ?LoggerInterface $logger = null,
+    ) {
+    }
+
+    public function onMessageFailed(WorkerMessageFailedEvent $event): void
+    {
+        /** @var ErrorDetailsStamp $errorStamp */
+        foreach ($event->getEnvelope()->all(ErrorDetailsStamp::class) as $errorStamp) {
+            if (RabbitMQTransportException::class === $errorStamp->getExceptionClass()) {
+                $this->logger?->info('[Keepalive] Skip message with RabbitMQTransportException from retry or critical log.');
+
+                return;
```

Comment:
loguję i przerywam na retry i niżej na failure,

analogicznie to było ze starym sposobem: https://github.com/answear/messenger-heartbeat-bundle/blob/c5d58b1e13f79b6c547acafc436652ffa5df2754/src/EventSubscriber/SendFailedMessageToFailureTransportListener.php

Niestety tak musi być w przypadku tych PRECONDIDTION 🤷🏻‍♂️ 😞 To, że one lecą to bardziej problemy serwerowe.